README.md

```diff
@@ -115,7 +111,7 @@ npx vercel link
 
 </details>
 
-## Step 2. Run Next.js locally in development mode
+## Step 3. Run Next.js locally in development mode
 
 When working locally you can make changes to your blog and studio, and your changes are applied live using hot reloading.
```

Comment:
The order of paragraphs doesn't seem logical to me here. I would suggest to first instract how to run the Studio locally and then say that your changes are applied live as you type

Reply:
Changed order 👍